README.md

```diff
@@ -74,7 +80,7 @@
 3. Run the following commands to setup the containers. This will setup the volumes required for node_modules and mongodb database. Also it will setup the containers for nodejs and mongodb.
     ```
     cd issue-tracker
-    tools/setup/setup
+    tools/setup/setup full
```

Comment:
Sometimes additional permissions are required to run these scripts. In my case, i had to run `chmod +x <filename>`. Can you mention that too, please? So if someone else has the same problem, he/she might find it helpful

Reply:
I have already done that in the `README` in the linux installation step 1.